Model/OrderData.php

```diff
@@ -32,4 +32,17 @@ public function getOrderChunks($storeId)
         $totalOrders = $this->getOrderQuery($storeId)->getSize();
         return (int) ceil($totalOrders / $this->chunkItems);
     }
+    
+    public function getDeletedProducts($storeId) {
```

Comment:
If you agree to move the deleted products to separate stage then it would make sense to have a separate data class for them, too. What do you think?

Reply:
Will be fixed with the next commit, still working on it.